preview-docs/Webhooks.md

```diff
@@ -0,0 +1,66 @@
+# RFC: Webhooks for schema change notifications
+
+To help you better integrate Apollo Studio into your unique workflow, we are starting some work to offer standard webhooks for certain events. The first of those events will be schema changes - any time we detect a change to the schema for your graph.
+
+We are looking for your feedback!
+
+If you have a related need that is not captured below, or if you see things we might be missing or problematic, please engage with us and ask questions through issues on this repository.
+
+## Plan restrictions
+
+Webhooks are available only to customers on our Enterprise plan.
+
+## How this works
+
+We will create a new integration channel: Webhooks.
```

Comment:
Some might not know what an "integration channel" is -- perhaps start with "On the Integrations page of your graph..." or something that situates them on what this is.